src/interface/obsidian/src/search_modal.ts

```diff
@@ -106,7 +106,7 @@ export class KhojSearchModal extends SuggestModal<SearchResult> {
 
         // Combine markdown and PDF results and sort them by score
         let results = mdData.concat(pdfData)
-            .sort((a: any, b: any) => b.score - a.score)
+            .sort((a: any, b: any) => a.score - b.score)
```

Comment:
I think we can remove this & make the query return `All`, rather than pdf/markdown separately. The results should come sorted on the server side.

Reply:
Yeah, valid point. 
It was an initial crude attempt to only show content from current vault. But this hasn't scaled well as other clients also support indexing PDF, Markdown file now. 

I've removed doing search queries for just markdown, pdf content for now.

A more proper solution, for later, would be a user configurable property to limit results to current vault and then actually limiting it via file/directory filters on server/client side.